WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -265,6 +267,15 @@ struct BlazeCampaignCreationForm: View {
             }
         }
     }
+
+    private var tosCheckboxToggle: some View {
+        Toggle(isOn: $viewModel.isToSAccepted) {
+            Text(viewModel.tosCheckboxAttributedText)
+                .bodyStyle()
+                .multilineTextAlignment(.leading)
```

Comment:
Nit: the external link should be displayed in a web view inside the app like [how we did](https://github.com/woocommerce/woocommerce-ios/blob/a9a1a4b156c991889779521f45e176f56f944ea4/WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift#L246-L250) on the payment confirm view. I'll add the same fix for the UPS terms view.